src/services/device-management/application.service.ts

```diff
@@ -450,15 +415,20 @@ export class ApplicationService {
         return orderBy;
     }
 
-    private getSortingForApplications(
-        query: ListAllEntitiesDto
-    ): Record<string, string | number> {
+    private getSortingForApplications(query: ListAllEntitiesDto): Record<string, string | number> {
         const sorting: Record<string, string | number> = {};
         if (
+            // TODO: Make this nicer
```

Comment:
Should this be removed or is it for the future? :-)

Reply:
Future, a bit more research for a nice way of doing it if it is possible